halyard/hal-scripts/user-scripts.tar

```diff
@@ -0,0 +1 @@
+This isn't a real tar file and sits as a placeholder to make terraform validate work
```

Comment:
There was no other way around this?

Reply:
using the filebase64 function to read files requires a file exists at the beginning of the terraform run. I can't think of a better way to get around this even though it is a little ugly.